selfdrive/ui/replay/Unlogger.cpp

```diff
@@ -127,8 +115,14 @@ void Unlogger::process() {
         last_elapsed = tc;
       }
 
-      auto e = *eit;
-      auto type = e.which();
+      cereal::Event::Reader e = *eit;
+
+      capnp::DynamicStruct::Reader e_ds = static_cast<capnp::DynamicStruct::Reader>(e);
+      std::string type;
+      KJ_IF_MAYBE(e_, e_ds.which()){
```

Comment:
is this necessary?

Reply:
i think this is the default way to remove the Maybe prefix